webplugin/js/app/mck-app.js

```diff
@@ -310,6 +310,15 @@ function ApplozicSidebox() {
             var widgetSettings = data.chatWidget;
             var disableChatWidget = options.disableChatWidget != null ? options.disableChatWidget : widgetSettings.disableChatWidget; // Give priority to appOptions over API data.
             
+            var allowedDomains = widgetSettings.allowedDomains;
+            var host = parent.window.location.host;
+
+            // Remove scripts if chatwidget is restricted by domains
+            if (Array.isArray(allowedDomains) && allowedDomains.length && !allowedDomains.includes(host)){
```

Comment:
will it work for sub domains as well, eg: if company have configured lets say epic.com and loads script in their multiple sites such as:
blog.epic.com
www.epic.com
dashboard.epic.com

Reply:
Each subdomain will have to be configured.

Reply:
is it possible to make this code flexible to cover this use case?

Reply:
Fixed